RtspServer/ClientSession.cs

```diff
@@ -393,16 +390,15 @@ public void StartReceive()
 
             if (object.ReferenceEquals(m_RtspSocket, null).Equals(false) && HasRuningServer)
             {
-                if (LastRecieve == null)
+                var receiveArgs = new SocketAsyncEventArgs
```

Comment:
If this really fixed the issue for you then try to just use a single SocketAsyncEventArgs which is disposed when the Session is disposed, you might have to refactor the server or client session to have a jump method which can switch out based on SocketAsyncEventArgs.SocketOperation to determine if was a send or receieve.